docs/content/v2024.2/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -12,6 +12,65 @@ menu:
+
+### How it works
+
+Automatic master failover works as follows:
+
+1. When active, by default YugabyteDB Anywhere checks universe masters every minute to see if they are healthy.
+
+    You can customize this interval using the universe runtime configuration option `yb.auto_master_failover.detect_interval`.
+
+1. YugabyteDB Anywhere declares a master is failed or potentially failed when any of the following conditions are met:
+
+    - The Master heartbeat delay is greater than the threshold.
+    - Maximum tablet follower lag is greater than the threshold.
+
+1. When YugabyteDB Anywhere detects an unhealthy master in a universe, it displays a message on the universe **Overview** indicating a potential master failure, and indicating the estimated time remaining until auto failover.
+
+    The warning is displayed when a master lags more than the threshold defined by the universe runtime configuration option `yb.auto_master_failover.master_follower_lag_soft_threshold`.
+
+    You can configure the time to failover using the universe runtime configuration option `yb.auto_master_failover.master_follower_lag_hard_threshold`.
+
+1. During this time, you can investigate and potentially fix the problem.
+
+    If you fix the problem, the warning is dismissed, and YugabyteDB Anywhere returns to monitoring the universe.
+
+    If you need more time to investigate or fix the problem manually, you can opt to snooze the failover.
+
+1. Failover is triggered if the time expires and the issue hasn't been fixed.
+
+    For a universe to successfully fail over masters, the following must be true:
+
+    - The universe is not paused.
+    - The universe is not locked (that is, another locking operation is running).
+    - All nodes are live; that is, there aren't any stopped, removed, or decommissioned nodes.
+
+    Note that master failover may not fix all the issues with the universe. Be sure to address other failed or unavailable nodes or other issues to bring your universe back to a healthy state.
+
+1. After starting up a new master on a different node in the same availability zone as the failed master, YugabyteDB Anywhere waits for you to recover any failed VMs, including the failed master VM, so that it can update the master address configuration on those VMs. Follow the steps in [Replace a live or unreachable node](#replace-a-live-or-unreachable-node).
+
+    You can set the delay for post automatic master failover using the universe runtime configuration option `yb.auto_master_failover.sync_master_addrs_task_delay`. The reference start time is calculated from the time that YugabyteDB Anywhere finds that all processes are running fine on all the VMs.
+
+For master failover, if the task fails, a retry is made automatically. The retry limit is set by the universe runtime configuration option `yb.auto_master_failover.max_task_retries`. Post failover, there is no retry limit as it is a critical operation.
```

Comment:
- So a user can set the retry limit using the flag if the master failover fails the first time. Correct? 
- Post failover, there is no retry limit as it is a critical operation - does it mean, a user cannot set the retry limit with the flag post failover if it successfully completed and there is actually no need to use the retry limit anymore? Unclear here ..